app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -58,6 +58,7 @@ module.exports = function (defaults) {
     autoImport: {
       forbidEval: true,
       insertScriptsAt: "ember-auto-import-scripts",
+      watchDependencies: ["discourse-i18n"],
```

Comment:
I was told on today's meeting that this apparently doesn't do anything in Embroider. I could have sworn that it does work? Anyway there is no harm in keeping this for now, but I'll double check later

Reply:
I did test making a change to the discourse-i18n/index.js file and it was reflected immediately, so *something* is working. not sure if this line affects it though.

Reply:
Yeah so this is probably not needed post-embroider, it seems like the v1 addon shim is what's actually causing the watching to happen